applications/databrowser/databrowser-plugins/org.csstudio.trends.databrowser2/META-INF/MANIFEST.MF

```diff
@@ -32,15 +32,15 @@ Require-Bundle: org.csstudio.utility.test;resolution:=optional,
  org.csstudio.utility.singlesource;bundle-version="1.0.0",
  org.csstudio.openfile;bundle-version="3.0.0",
  org.diirt.datasource-vtype;bundle-version="3.0.1",
- org.csstudio.utility.pvmanager.loc;bundle-version="3.1.0";resolution:=optional,
- org.csstudio.utility.pvmanager.sim;bundle-version="3.1.0";resolution:=optional,
  org.csstudio.autocomplete.ui,
  org.csstudio.archive.vtype;bundle-version="2.0.0",
  org.csstudio.email;bundle-version="1.2.0",
  org.diirt.util;bundle-version="3.0.1",
  org.diirt.vtype;bundle-version="3.0.1",
  org.diirt.datasource;bundle-version="3.0.1",
- org.diirt.datasource-extra;bundle-version="3.0.1"
+ org.diirt.datasource-extra;bundle-version="3.0.1",
```

Comment:
@kasemir I removed utility.pvmanager.loc and sim (optional) and replaced them with diirt.datasource (optional)